### ISSUE TITLE
ProgressDialog.show method should behave like a stack

### ISSUE THREAD

**Issue by admtriggo** [CruxFramework/crux#91]
_From [gessedafe@gmail.com](https://code.google.com/u/gessedafe@gmail.com/) on May 03, 2010 17:10:02_

For example, when a screen is blocked by two consecutives calls to the
method ProgressDialog.show, this screen only should be effectively
unblocked after a second call to the method ProgressDialog.hide.

_Original issue: http://code.google.com/p/crux-framework/issues/detail?id=91_

**Comment by admtriggo** [comment]
_From [gessedafe@gmail.com](https://code.google.com/u/gessedafe@gmail.com/) on May 03, 2010 13:14:20_

**Status:** Fixed

**Comment by admtriggo** [comment]
_From [tr_busta...@yahoo.com.br](https://code.google.com/u/115454294030253308352/) on May 04, 2010 09:40:58_

Issue 82 has been merged into this issue.

**Comment by admtriggo** [comment]
_From [tr_busta...@yahoo.com.br](https://code.google.com/u/115454294030253308352/) on May 12, 2010 15:23:20_

**Labels:** CruxWidgetLibrary

**Comment by admtriggo** [comment]
_From [tr_busta...@yahoo.com.br](https://code.google.com/u/115454294030253308352/) on May 12, 2010 15:24:45_

**Labels:** -CruxWidgetLibrary CruxWidgetsLibrary

**Comment by admtriggo** [comment]
_From [tr_busta...@yahoo.com.br](https://code.google.com/u/115454294030253308352/) on May 26, 2010 15:38:29_

**Status:** Verified
